_posts/2017-06-21-pragmatic-architecture-today.md

```diff
@@ -0,0 +1,280 @@
+  <img class="image fit" alt="System Integrity" src="/img/prag-arch/engine.jpg">
+</p>
+
+When creating a 10 million Eur car, many teams (with different responsibilities) are involved.
+The people who design the engine are different from the people who design the brakes.
+Creating the best engine, creating the best brakes, ... does not imply you will have the best car.
+Everything needs to work together.
+**The integrity of the system is very important.**
+This point is again proven by Formule One: other teams can buy the current Mercedes engine.
+They might win some races, but they haven't won the world championship
+
+> [Russell L. Ackoff](https://en.wikipedia.org/wiki/Russell_L._Ackoff)
+>
+> A system is more than the sum of its parts; it is an indivisible whole. 
+> It loses its essential properties when it is taken apart.
+
+To ensure system integrity, the software architect needs to be part of the project team.
+He must make sure that he enables the right people to collaborate on the system architecture.
+Being part of the team does not mean not taking **responsibility**.
+It is great to give **ownership** to members of the team, but in the end the architect needs to stay accountable.
+When collaborating, an architect should not enforce all technical decisions.
+Part of working as a team, is accepting to be challenged and embracing this challenge.
+When team members have spirited discussions, it shows they are passionate enough about their craft to care.
+Disagreeing and discussing alternatives is a great way to come to a better solution and to learn from each other.
+Being part of the team, as an architect, will lead to a system with a consistent vision, where the implementation matches the architectural description.
+This also implies that an architect should be involved in the codebase of the system: writing code, performing code-reviews, doing proof-of-concepts, supporting others, ...
+By being involved in the codebase, you can make sure that the architectual description is understood by the developers.
+
+## Visual?
+
+<p style="text-align: center;">
+  <img class="image fit" alt="Visual" src="/img/prag-arch/visual.jpg">
+</p>
+
+While code is a very important asset of a working system, code alone is not enough to have an accurate and lasting description of a system.
+
+> [Grady Booch](https://nl.wikipedia.org/wiki/Grady_Booch)
+>
+> One cannot see the ocean's currents by studying drops of water.
+
+The goal of visually representing a system, through the architectural description, is to make sure the architecture of the system is in the stakeholder's heads.
+The visual representation can be used to check for consistency,  reporting, validation and **sharing information**.
+
+### Some ground rules
+
+While UML has its merits, often it is not necessary to create an extensive UML model for the architecture.
+It will be time-consuming and, unfortunately, often UML is not correctly understood by stakeholders.
```

Comment:
Would rephrase this a bit:
It will be time-consuming and, unfortunately, **it is often the case that** UML is not ...